modules/tinymce/src/themes/silver/test/ts/browser/editor/core/ChoiceControlsTest.ts

```diff
@@ -231,28 +240,24 @@ describe('browser.tinymce.themes.silver.editor.core.ChoiceControlsTest', () => {
     });
 
     context('Advanced settings', () => {
-      beforeEach(() => {
-        hook.editor().settings.content_langs = undefined;
-      });
+      // Approximate the BDD hook here, but unfortunately we need a different editor per test
+      let editor: Editor | null = null;
```

Comment:
This isn't how we do this anywhere else in the tests. We always just call `McEditor.remove` at the end to avoid this mess, so can you please change that to keep things consistent?

Reply:
I put it outside in an `after` so that it'd get called even if the test failed / an exception was thrown. Do we want the editor to persist in the case of a failure?

Reply:
Yes, we do the same thing in `TinyHooks` so that you can inspect and debug the error.